backend/src/models/Category.js

```diff
@@ -0,0 +1,21 @@
+import uuid from 'uuid/v4'
+
+module.exports = {
+  id: { type: 'string', primary: true, default: uuid },
+  name: { type: 'string', required: true, default: false },
+  slug: { type: 'string' },
+  icon: { type: 'string', required: true, default: false },
+  createdAt: { type: 'string', isoDate: true, default: () => new Date().toISOString() },
+  updatedAt: {
+    type: 'string',
+    isoDate: true,
+    required: true,
+    default: () => new Date().toISOString(),
+  },
+  post: {
+    type: 'relationship',
+    relationship: 'CATEGORIZED',
+    target: 'Post',
+    direction: 'in',
+  },
+}
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-21T14:18:41Z" title="Wednesday, August 21st 2019, 4:18:41 pm +02:00">Aug 21, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,21 @@
+import uuid from 'uuid/v4'
+
+module.exports = {
+  id: { type: 'string', primary: true, default: uuid },
+  name: { type: 'string', required: true, default: false },
+  slug: { type: 'string' },
+  icon: { type: 'string', required: true, default: false },
+  createdAt: { type: 'string', isoDate: true, default: () => new Date().toISOString() },
+  updatedAt: {
+    type: 'string',
+    isoDate: true,
+    required: true,
+    default: () => new Date().toISOString(),
+  },
+  post: {
+    type: 'relationship',
+    relationship: 'CATEGORIZED',
+    target: 'Post',
+    direction: 'in',
+  },
+}
```


Love it :heart_eyes: